Demo/src/main/java/com/paypal/android/api/services/PayPalApi.kt

```diff
@@ -18,6 +19,7 @@ interface PayPalApi {
     @Headers("Accept: application/json")
     @POST("/v2/vault/payment-tokens")
     suspend fun postApprovalSessionId(
+        @Header("Authorization") accessToken: String,
```

Comment:
Q: why is this needed?

Reply:
`v2/vault/payment-tokens` requires and Access Token to work